otterdog/eclipse-ibeji.jsonnet

```diff
@@ -21,8 +21,15 @@ orgs.newOrg('eclipse-ibeji') {
       allow_merge_commit: true,
       allow_update_branch: false,
       delete_branch_on_merge: false,
+      description: "freyja project",
       secret_scanning_push_protection: "disabled",
```

Comment:
Either this PR or a separate one, but 'secret_scanning_push_protection' should either be removed and/or enabled.